docs/operations/common_configuration/zone_aware_routing.rst

```diff
@@ -0,0 +1,13 @@
+.. _common_configuration_zone_aware_routing:
+
+Zone aware routing
+==================
+
+Do the following if you want to enable :ref:`zone aware routing <arch_overview_load_balancing_zone_aware_routing>`
+between cluster A and cluster B:
+
+* Both cluster A and cluster B must be using :ref:`discovery service <arch_overview_service_discovery_sds>` with
```

Comment:
Can you add a sample of what the cluster entry would be? And what other config changes with direct link to doc?